test/function/namespacing-collisions-2/main.js

```diff
@@ -0,0 +1,7 @@
+import { Something } from './Something';
+import { SomethingAgain } from './SomethingAgain';
+import { Material } from './Material';
+import { MaterialAgain } from './MaterialAgain';
+
+var result = [Material(), MaterialAgain(), Something(), SomethingAgain()]
```

Comment:
Since rollup supports 0.12 (again) we have to use buble on anything that doesn't run natively. Most of the tests use `var` if I remember correctly.